### DISCUSSION TITLE
Is Shairport Sync  / Apple Airplay protocol supported on pipewire?

### DISCUSSION THREAD

**Discussion by alexbramford** [mikebrady/shairport-sync#1271] [Q&A] [answered]
**Use case:**

- I'm looking to upgrade from Fedora 32 to Fedora 34
- And pipewire is the replacement for pulseudio on F34

**Setup:**

- F32(Rhythmbox) --> pulseaudio (module-raop-sink) --> LAN--> RaspberryPi (moodeAudio + Shairport Sync)

**Question:**

- Is Shairport Sync supported by pipewire?

**Specifically:**

1. pulseaudio  **module-raop-sink** on my Fedora 32 laptop
2. shairport-sync running on the RaspberryPi

Or is there a pipewire module that will do the same as pulseaudio (module-raop-sink?)

**References:**

- https://www.freedesktop.org/wiki/Software/PulseAudio/Documentation/User/Modules/#module-raop-sink

- https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/1542

(If this not the appropriate forum, please recommend the correct place to ask this question!)

**Comment by mikebrady** [comment, marked as answer]
This was moved to Issue 1219 and answered.
